### ISSUE TITLE
macOS is wrongly identified as Windows

### ISSUE THREAD

**Issue by 5amch3n** [codefeathers/rollup-plugin-svelte-svg#2]
```javascript
// rollup-plugin-svelte-svg.cjs.js line #10 (src/index.js line #7)
const isWindows = os.platform().includes("win");
```

On macOS, os.platform() returns 'darwin', which includes 'win'. When macOS is wrongly identified as Windows, an error occurs, something like "[!] (plugin svg) Error: options.name must be a valid identifier (got '/Users/samch3n/my-project/static/svgs/solid/book')"

**Comment by MKRhere** [comment]
Good find, thanks. I'll fix this in the next release.

**Comment by MKRhere** [comment]
Fixed in ddd54aa3b46bc08843473c08150d09b77ad7df30
v0.2.1: [npm](https://www.npmjs.com/package/rollup-plugin-svelte-svg/v/0.2.1)
